code/src/library/Containers.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components'
+
+export const Main = styled.main`
+${({noList}) => noList?`height: 100vh;`:`height:100%;`}
+background-image: linear-gradient(45deg, #4e6048, #70836d, #93a895, #b8cebf);
+display: grid;
+grid-template-column: 1fr;
```

Comment:
No need to set grid-template-column, since this will be per default depending on the structure of the code below.